website/docs/docs/use-dbt-semantic-layer/exports.md

```diff
@@ -0,0 +1,179 @@
+
+When you run a build job, any saved queries downstream of the dbt models in that job will run as well. To make sure your Export data is up-to-date, run the Export as a downstream step (after the model). The steps to create an Export are:
+
+1. Create a [deploy job](/docs/deploy/deploy-jobs) in dbt Cloud.
+2. Add the `--include-saved-query` to the `dbt build` command in your **Execution Settings** to re-run any Export that needs to be refreshed after a model is build. 
+   - You can use the [selector syntax](/reference/node-selection/syntax) `--select` to only run the exports downstream of a particular model.
+3. After dbt finishes building the models, the MetricFlow Server processes the exports, compiles the necessary SQL, and executes this SQL against your data platform.
+4. Review the exports execution details in the jobs logs and confirm the Export was run successfully. Since saved queries are integrated into the dbt DAG, all outputs related to exports are available in the job logs.
+5. Your data is now available in the data platform for querying.
+
+<Lightbox src="/img/docs/dbt-cloud/semantic-layer/deploy_exports.jpg" width="90%" title="Adding --include-saved-query to the dbt build command in your job execution settings." />
+
+You can use the selector syntax `--select` or `-s` to specify a particular dbt model to run in your build command in order to only run the exports downstream of that model. As an example, the following command will run any saved queries that are downstream of the `orders` semantic model:
+
+```bash
+dbt build --include-saved-query --select orders+
+```
+
+## FAQs
+
+<detailsToggle alt_header="Can I have multiple exports in a single saved-query?">
+
+Yes, this is possible. However, the only difference would be the name, schema, and materialization strategy of the Export.
+</detailsToggle>
+
+<detailsToggle alt_header="How do I run all exports for a saved query?">
+
+- In production runs, you build a model and any exports downstream of that model. There is currently no way to call the Export directly from the Job scheduler.
+- In development, you can run all exports by running `dbt sl Export --saved-query sq_name`.
+
+</detailsToggle>
+
+<detailsToggle alt_header="Will I run duplicate exports if multiple models are downstream of my saved query?">
+
+We will only run each Export once even if we build multiple models that are downstream of the saved query. For example, you may have a saved query called `order_metrics`, which has metrics from both the `orders` and `order_items` semantic models. 
+
+You can run a job that includes both models: `dbt build --include-saved-query`. This runs both the `orders` and `order_items` models, however it will only run the `order_metrics` Export once.
```

Comment:
sugg: keep example together 

```suggestion
For example, you might have a saved query called `order_metrics`, which has metrics from both the `orders` and `order_items` semantic models. You can run a job that includes both models: `dbt build --include-saved-query`. This runs both the `orders` and `order_items` models, however it will only run the `order_metrics` export once.
```